packages/dso-toolkit/src/components/label/label.mixins.scss

```diff
@@ -81,8 +73,12 @@
   &.dso-label-bright {
     background-color: label-variables.$bright-bg-color;
     color: label-variables.$bright-color;
-    border: 1px solid label-variables.$bright-border-color;
-    padding: #{units.$u1 * 0.5 - 1px} units.$u1 - 1px;
+    outline: 1px solid label-variables.$bright-border-color;
```

Comment:
Hier is dit OK, ik ben zelf voorstander van het gebruik van box-shadow. Met outline kun je focus-stijling van de browser overschrijven.